definitions/features/instance.rb

```diff
@@ -1,8 +1,13 @@
+require 'net/http'
+require 'json'
+
 class Features::Instance < ForemanMaintain::Feature
   metadata do
     label :instance
   end
 
+  attr_reader :last_ping_failing_services, :last_ping_status, :last_ping_result
```

Comment:
Perhaps we could encapsulate this three into one class and make the `ping` method return it, if that's what we need to use it outside of the `ping?` method

Reply:
I decided not to go this way originally but I agree it does not look that well in the end. Updated.